src/components/UserPortal/CreateGroupChat/CreateGroupChat.tsx

```
@@ -206,50 +203,51 @@ export default function CreateGroupChat({
+            <FormControl fullWidth>
+              <InputLabel id="select-org">Select Organization</InputLabel>
+              <Select
+                labelId="select-org"
+                id="select-org"
+                data-testid="orgSelect"
+                label="Select Organization"
+                value={selectedOrganization}
+                onChange={handleChange}
               >
                 {organizations &&
                   organizations.length &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)